views/admin/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -28,6 +28,10 @@
       display: inline-block;
       margin-left: 12px;
     }
+
+    .pg-multiselect {
```

Comment:
our prefix id `wd` - can you change that please, also for the other selectors!?

Reply:
Done 👍